ncat/ncat_connect.c

```diff
@@ -695,21 +695,18 @@ static int do_proxy_socks5(void)
 
     if (send(sd, (char *) &socks5msg, len, 0) < 0) {
         loguser("Error: proxy request: %s.\n", socket_strerror(socket_errno()));
-        close(sd);
-        return -1;
+	goto error;
```

Comment:
All the ```goto error;``` statements are indented by tabs whereas the surrounding code uses spaces for indentation.